sdk/lusid/utilities/api_client_factory.py

```diff
@@ -51,6 +52,11 @@ def __init__(self, **kwargs):
         # Otherwise use a secrets file if it exists
         builder_kwargs["api_secrets_filename"] = kwargs.get("api_secrets_filename", None)
 
+        # add the correlation id if specified
+        corr_id = kwargs.get("correlation_id", None)
+        if corr_id is not None:
+            builder_kwargs["correlation_id"] = corr_id
```

Comment:
Can probably just pass it through as None rather than bothering with this check as it will be handled later. We do the same with the secrets filename just above.

Reply:
done